app/posts/register-trainee-teachers/2022-11-01-when-everythings-gone-wrong.md

```diff
@@ -0,0 +1,64 @@
+---
+title: When everything’s gone wrong
+description: A look at our ‘maintenance page’ - a page to be used when the service is offline
+date: 2022-11-01
+related:
+  items:
+  - text: Pull request updating our maintenance page
+    href: https://github.com/DFE-Digital/register-trainee-teachers/pull/2880
+  - text: Register’s maintenance page code
+    href: https://github.com/DFE-Digital/register-trainee-teachers/tree/main/service_unavailable_page
+ogImage:
+  src: /register-trainee-teachers/when-everythings-gone-wrong/1.maintenance-page-screenshot.png
+  alt: Screenshot of Register’s new maintenance page
+---
+
+![Screenshot of Register’s new maintenance page.](1.maintenance-page-screenshot.png)
+
+This post is about our ‘maintenance page’ - colloquially known as our ‘everything’s gone wrong page’. This is a page to be used when the entire service is down - either for maintenance or because something has gone very wrong with Register trainee teachers (Register). 
```

Comment:
```suggestion
This post is about our ‘maintenance page’ - known as our ‘everything’s gone wrong page’ within our team. This is a page to be used when the entire service is down - either for maintenance or because something has gone very wrong with Register trainee teachers (Register). 
```